src/services/DogApi.js

```diff
@@ -0,0 +1,40 @@
+const myHeaders = new Headers();
+
+myHeaders
+  .append('Authorization', 
+    'Bearer 1544caee-0010-41ae-92cb-28cbee4c0e70');
```

Comment:
This is awesome! But....I can now steal your token (or anyone since it's on Github now). Refresh your token and put the new one in a .env file for safety!